bank-sdk/example-app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    package="net.gini.android.capture.screen">
+    package="net.gini.android.bank.screen">
```

Comment:
[Request] I looked at the Health SDK's example app and it's package name is `net.gini.android.health.sdk.exampleapp`. Let's follow that pattern and use `net.gini.android.bank.sdk.exampleapp` package name for the Bank SDK's example app.